src/lib/browser/Adblocker.ts

```diff
@@ -0,0 +1,33 @@
+import { request } from 'undici';
+
+const list = 'https://badmojr.github.io/1Hosts/Lite/domains.txt';
```

Comment:
The list is big (1.3Mo), so it will probably prevent many requests but will also add quite some bandwidth usage.
Any reason to not embed it in the image? It will become outdated overtime but as discussed, the main domains never change, and we can update it at each new release.

Reply:
Indeed, once at startup sounds good 👍